lib/flag_shih_tzu.rb

```diff
@@ -242,118 +280,121 @@ def flag_keys(colmn = DEFAULT_COLUMN_NAME)
 
     private
 
-      def flag_value_range_for_column(colmn)
-        max = flag_mapping[colmn].values.max
-        Range.new(0, (2 * max) - 1)
-      end
+    def flag_value_range_for_column(colmn)
+      max = flag_mapping[colmn].values.max
+      Range.new(0, (2 * max) - 1)
+    end
 
-      def chained_flags_values(colmn, *args)
-        val = flag_value_range_for_column(colmn).to_a
-        args.each do |flag|
-          neg = false
-          if flag.to_s.match /^not_/
-            neg = true
-            flag = flag.to_s.sub(/^not_/, '').to_sym
-          end
-          check_flag(flag, colmn)
-          flag_values = sql_in_for_flag(flag, colmn)
-          if neg
-            val = val - flag_values
-          else
-            val = val & flag_values
-          end
+    def chained_flags_values(colmn, *args)
+      val = flag_value_range_for_column(colmn).to_a
+      args.each do |flag|
+        neg = false
+        if flag.to_s.match /^not_/
+          neg = true
+          flag = flag.to_s.sub(/^not_/, '').to_sym
         end
-        val
-      end
-
-      def parse_flag_options(*args)
-        options = args.shift
-        if args.size >= 1
-          add_options = args.shift
+        check_flag(flag, colmn)
+        flag_values = sql_in_for_flag(flag, colmn)
+        if neg
+          val = val - flag_values
         else
-          add_options = options.keys.select {|key| !key.is_a?(Fixnum)}.inject({}) do |hash, key|
-            hash[key] = options.delete(key)
-            hash
-          end
+          val = val & flag_values
         end
-        return options, add_options
       end
+      val
+    end
+
+    def parse_flag_options(*args)
+      options = args.shift
+      if args.size >= 1
+        add_options = args.shift
+      else
+        add_options =
+          options.keys.
+            select {|key| !key.is_a?(Fixnum)}.
```

Comment:
Align the operands of an expression in an assignment spanning multiple lines.<br>Space between { and | missing.<br>Space missing inside }.